api/metrics-api-jaxrs/pom.xml

```diff
@@ -177,14 +176,13 @@
           <plugin>
             <groupId>com.github.kongchen</groupId>
             <artifactId>swagger-maven-plugin</artifactId>
-            <version>2.3.3</version>
             <configuration>
               <apiSources>
                 <apiSource>
                   <locations>org.hawkular.metrics.api.jaxrs</locations>
                   <apiVersion>1.0</apiVersion>
                   <basePath>http://localhost:8080/hawkular-metrics/</basePath>
-                  <outputTemplate>${basedir}/src/main/resources/rest-doc/asciidoc.mustache</outputTemplate>
+                  <outputTemplate>https://raw.githubusercontent.com/hawkular/hawkular.github.io/swagger/asciidoc.mustache</outputTemplate>
```

Comment:
> Is that cached? Or how would I build the docs offline?

No cache, normal http get is performed. So when building with `-Pdocgen` in train it'd fail the build. Workaround is to download the file and enter the local path to it. 

I could have added another profile for that called `docgenOffline` with a link to the local file (per component), but it would defeat the purpose of the change -> having the standardized form of documentation.